wazo_call_logd/tests/test_generator_scenarios.py

```diff
@@ -772,3 +772,224 @@ def test_bad_apple(self, cels: list[CEL]):
+            CHAN_END                  | 1738482679.51 | 1738482670.36 | 2025-02-02 11:21:21.670455+03:30 | John Conner                | 0012345678910                        | p4go4oxa                             | wazo_wait_for_registration                                | Local/p4go4oxa@wazo_wait_for_registration-0000000b;2                  |                 |                                                                                                                                              |
+            HANGUP                    | 1738482679.53 | 1738482670.36 | 2025-02-02 11:21:21.670731+03:30 | John Conner                | 0012345678910                        | 447b36i8                             | wazo_wait_for_registration                                | Local/447b36i8@wazo_wait_for_registration-0000000c;2                  |                 |                                                                                                                                              | {"hangupcause":16,"hangupsource":"","dialstatus":""}
+            CHAN_END                  | 1738482679.53 | 1738482670.36 | 2025-02-02 11:21:21.670731+03:30 | John Conner                | 0012345678910                        | 447b36i8                             | wazo_wait_for_registration                                | Local/447b36i8@wazo_wait_for_registration-0000000c;2                  |                 |                                                                                                                                              |
+            CHAN_END                  | 1738482679.49 | 1738482670.36 | 2025-02-02 11:21:21.886307+03:30 | John Conner                | 0012345678910                        | s                                    | ctx-Company-internal-3344220c-6383-4581-9696-15055ec303e3 | PJSIP/uqe8wu8g-0000000b                                               | AppDial2        | (Outgoing Line)                                                                                                                              |
+            LINKEDID_END              | 1738482679.49 | 1738482670.36 | 2025-02-02 11:21:21.886307+03:30 | John Conner                | 0012345678910                        | s                                    | ctx-Company-internal-3344220c-6383-4581-9696-15055ec303e3 | PJSIP/uqe8wu8g-0000000b                                               | AppDial2        | (Outgoing Line)                                                                                                                              |
```

Comment:
Same comment as test above, scenario should be simplified to produce a more minimal CEL output.